cli/tests/pcluster/validators/test_ec2_validators.py

```diff
@@ -603,86 +604,169 @@ def test_capacity_reservation_validator(
 mock_bad_config = {"GroupConfiguration": {"Configuration": [{"Type": "AWS::EC2::MockService"}]}}
 
 at_least_one_capacity_reservation_error_message = (
-    "Capacity reservation resource group .* must have at least "
-    "one capacity reservation for c5.xlarge in the same availability "
-    "zone as subnet subnet-123."
+    "Capacity reservation resource group .* must have at least one capacity reservation for c5.xlarge."
 )
```

Comment:
Optional: rather than a fix string, you may define a function that prints that string depending on input parameters (such as the CR resource group ARN or the instance type).